### ISSUE TITLE
Compile error

### ISSUE THREAD

**Issue by fabaff** [OttoWinter/esphomelib#2]
On fresh installation I tried to use `esphomelib` with the example but ended up with a compile error.

```bash
src/main.cpp:1:36: fatal error: esphomelib/application.h: No such file or directory
compilation terminated.
```

Sorry, I can't help to debug this any further because I'm not really PlatformIO user.

**Comment by OttoWinter** [comment]
Oh yes, sorry, that seems to be due to my incorrect setup of platformio (it's kind of hard to check whether the library works without publishing it first). Creating a fix right now - but including `application.h` (without `esphomelib`) instead should probably work too

**Comment by fabaff** [comment]
Sounds like a "Chicken-or-the-Egg" issue. I assume that the next error is related to the path as well.

```bash
Error: Could not find `ArduinoJSON` dependency for `esphomelib` library
```

I see that `ArduinoJSON` is available in the project.

**Comment by OttoWinter** [comment]
It seems to be fixed now, at least when manually recloning the library into the `.piolibdeps` folder - which should emulate platformio downloading the latest version. The platformio library crawler will probably take a day to update the registry though.

Thanks for reporting this! A library is always better when it's actually usable 😂 .

**Comment by OttoWinter** [comment]
Fixed with version 0.1.3 and library crawler has picked up the update.

**Comment by fabaff** [comment]
Both issue are gone in 0.1.3 . Thanks.
